src/wh_server_crypto.c

```diff
@@ -709,6 +747,12 @@ static int hsmCryptoCmac(whServerContext* server, whPacket* packet,
                 keyId = MAKE_WOLFHSM_KEYID(WOLFHSM_KEYTYPE_CRYPTO,
                     server->comm->client_id, packet->cmacReq.keyId);
             }
+#ifdef WOLFHSM_SEPARATE_BIG_CACHE
+            /* evict the aes sized key in the normal cache */
+            if (moveToBigCache == 1) {
+                ret = hsmEvictKey(server, keyId);
+            }
+#endif
```

Comment:
Not super clear why we are doing this? Could you explain plz

Reply:
if a user pre-caches an aes key to use with cmac it will be in the regular cache initially, when we cache the cmac struct it will need to go into big cache so the now defunct aes key will be hanging if we don't evict it